src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -138,6 +138,9 @@ def __init__(self, config):
         # These are added now by the condor client
         #self.x509userproxysubject = proxy.getSubject()
         #self.x509userproxyfqan = proxy.getAttributeFromProxy(self.x509userproxy)
+        
+        self.tc = TagCollector()
+        self.rel_microarchs = self.tc.defaultMicroArchVersionNumberByRelease()
```

Comment:
From what I see how this plugin is initialized, it looks like it would be initialized only once, upon JobSubmitter start up. 
If you agree with that assessment, then we need to evaluate this call to build the map of release vs micro-architecture as part of the component polling cycle, such that we keep updating that map. From what I can see, the `submit()` method would be a good place to keep it. Can you please investigate this further and fix it if needed?

Reply:
@amaltaro Done!